hr_employee_calendar_planning/views/hr_employee_views.xml

```diff
@@ -10,7 +10,10 @@
                 <attribute name="invisible">1</attribute>
             </field>
             <field name="resource_calendar_id" position="after">
-                <field name="calendar_ids" context="{'default_company_id': company_id}">
+                <field
+                    name="calendar_ids"
+                    context="{'default_company_id': parent.company_id}"
```

Comment:
Why the parent? It was removed on other PRs company_id is already a field from hr.employee so it should not be necessary.

You can check that it does not work following this steps:

- Create a company
- Select both companies on your view
- Create a new employee for company B and create a calendar from the widget. The calendar created will be from company A not B.

Reply:
Sorry, following the steps you indicate I see that it was not working correctly.

The change that I have applied does work, for example
![ejemplo-v15](https://user-images.githubusercontent.com/4117568/187709386-a35bd820-168b-47b6-b794-9d2379856ae8.gif)